MRETestBed/Assets/Scenes/FunctionalTest-localhost.unity

```diff
@@ -851,8 +851,9 @@ MonoBehaviour:
     type: 2}
   MREURL: ws://localhost:3901
   testNames:
-  - asset-early-assignment-test
+  - asset-early-assignment
```

Comment:
>asset-early-assignment [](start = 4, length = 22)

Please have each test end in 'test', to match the test names in app.ts.

Reply:
I just noticed that app.ts actually has it without test too. I can rename that instead if we want.

Reply:
Let's have every test end in '-test'.